src/PortfolioPlanning/Components/Plan/DependencyPanel.tsx

```diff
@@ -135,7 +139,10 @@ export class DependencyPanel extends React.Component<IDependencyPanelProps, IDep
         }
     }
 
-    private _renderDependencyGroup = (dependencies: PortfolioPlanningQueryResultItem[]): JSX.Element => {
+    private _renderDependencyGroup = (
+        dependencies: PortfolioPlanningQueryResultItem[],
+        dependsOn: boolean

```

Comment:
I think it will be easier to understand if we use Predecessor and Successor like Ed will be using. `predecessorGroup` or something.

Reply:
I have renamed all `DependsOn `and `HasDependency `to `predecessors `and `successors`.